.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,14 +8,14 @@ These are the most common things requested on pull requests (PRs).
 
 Remember that PRs should be made against the dev branch, unless you're preparing a pipeline release.
 
-Learn more about contributing: [CONTRIBUTING.md](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
+Learn more about contributing: [CONTRIBUTING.md](https://github.com/nf-core/sarek/tree/main/.github/CONTRIBUTING.md)
```

Comment:
So if we only mention `main` here, we have to switch all over the place, no?

Reply:
this is now master because the default branch was master, once you change `defaultBranch` from the `manifest` and run `nf-core pipelines sync` all of these mentions will change to `main`.

Reply:
actually, sorry, this was not true, the default branch is automatically detected from the git configuration :)